src/firmware/main/main.c

```diff
@@ -48,14 +49,14 @@
 #include "io/feedback.h"
 #include "io/icb.h"
 #include "io/leds.h"
+#include "io/lps.h"
 #include "io/motor.h"
 #include "io/mrf.h"
 #include "io/pins.h"
 #include "io/receive.h"
 #include "io/sdcard.h"
 #include "io/usb_config.h"
 #include "io/wheels.h"
```

Comment:
same as above

Reply:
Similar comment to above, out-of-scope for this PR.